src/hyperfine/benchmark.rs

```diff
@@ -39,23 +39,29 @@ fn subtract_shell_spawning_time(time: Second, shell_spawning_time: Second) -> Se
 pub fn time_shell_command(
     command: &Command,
     failure_action: CmdFailureAction,
+    capture_out: bool,
     shell_spawning_time: Option<TimingResult>,
 ) -> io::Result<(TimingResult, bool)> {
     let wallclock_timer = WallClockTimer::start();
 
-    let result = execute_and_time(&command.get_shell_command())?;
+    let result = execute_and_time(&command.get_shell_command(), capture_out)?;
 
     let mut time_user = result.user_time;
     let mut time_system = result.system_time;
 
     let mut time_real = wallclock_timer.stop();
 
     if failure_action == CmdFailureAction::RaiseError && !result.status.success() {
-        return Err(io::Error::new(
-            io::ErrorKind::Other,
-            "Command terminated with non-zero exit code. \
-             Use the '-i'/'--ignore-failure' option if you want to ignore this.",
-        ));
+        let errstring = if capture_out {
+            format!("Command terminated with non-zero exit code. \
+                 Use the '-i'/'--ignore-failure' option if you want to ignore this.\nStdout: {}\nStderr: {}", result.stdout, result.stderr)
```

Comment:
Maybe `Stdout` and `Stderr` should be more prominent in the output (maybe all-caps `STDOUT` and a newline after the `:`)?